addon/-private/helpers.js

```diff
@@ -309,3 +313,37 @@ export function getProperty(object, pathToProp) {
 }
 
 export const assign = Ember.assign || Ember.merge;
+
+/**
+ * @public
+ *
+ * Fills an input, textarea, or contenteditable element with the passed-in content.
+ *
+ * @param {$Element} $element             jQuery DOM Element
+ * @param {string} content                Content to be inserted into the element
+ * @param {Object} options                Options for error reporting in the case
+ * where this helper is used within an execution context
+ * @param {string} options.selector       Element selector
+ * @param {Ceibo} options.pageObjectNode  PageObject node containing the method
+ * which, when invoked, resulted in this call to `fillElement`
+ * @param {string} options.pageObjectKey  Key of method on PageObject
+ * which, when invoked, resulted in this call to `fillElement`
+ * @return
+ *
+ * @throws Will throw an error if called on a contenteditable element that has
+ * `contenteditable="false"`
+ */
```

Comment:
Is this level of documentation right for this kind of helper? Is it correct to mark it `@public` because it is imported by the execution contexts, even though it is not exposed in the actual add-on exports? Now that this is in `helpers.js`, does it need its own unit test?